markdown/events/2022/bytesize-inkscape.md

```diff
@@ -25,3 +25,93 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[3:34](https://www.youtube.com/watch?v=0vKhfedYKGo&t=214)
+You can also make workflow diagrams. This is one from Eager. This is a more a general, broad overview, which helps people to understand what are the major components of the pipeline and what it does. It doesn't go into too much detail and these are the sections and the tools it uses. If you want something more detailed, you can make something like this, which shows more the direct connections between the different components, the different tools of the particular pipeline. These give you two different purposes: This is more for selling the pipeline, if you want to pique someone's interest, by for example putting by it in a publication. Whereas this is more for the user who's actually going to be interacting with the pipeline, trying to run it. There are different stages and levels of such pipeline diagrams. You can use them to help make your pipeline more accessible from the concept of what it does.
+This format is the one that became quite popular recently, but I've been playing around with other variants. This is a new one, that I'm coming up with for nf-core/taxprofiler pipeline. This shows that you don't have to stick with the particular one that I showed in a previous slide, you can play around with this as well.
+
+[4:51](https://www.youtube.com/watch?v=0vKhfedYKGo&t=291)
+All of these diagrams, all these images are done in Inkscape. In addition, it doesn't have to just be diagrams to show what you can do in your documentation. One thing that we did with Eager, which was also quite popular and I was quite proud of, is that my co-author Sandra Wegener, she drew these very nice schematics cartoony diagrams for the output documentation. This helps people to understand better what they should be looking for when they're doing quality control in their pipeline runs. For example, she took what you get from MultiQC, so for example FastQC in this case, the first QC section. She drew little cartoony versions of what you should be seeing, separated by the three different boxes, but with little notes saying exactly what you are seeing in this and how you should interpret the results here. This acts as a very nice, quick and fast reference for people, to understand what your pipeline has done at the other end. Also it adds a little bit of fun, a little bit of color, to otherwise often very dry documentation, that can be often be very dull. Particularly for very big pipelines like Eager and Sarek, and when you have a lot of output documentation, such things can break things up a little bit and make it more accessible and more fun for a user to use.
```

Comment:
```suggestion
All of these diagrams, all these images are done in Inkscape. In addition, it doesn't have to just be diagrams to show what you can do in your documentation. One thing that we did with Eager, which was also quite popular and I was quite proud of, is that my co-author Zandra Fagernäs she drew these very nice schematics cartoony diagrams for the output documentation. This helps people to understand better what they should be looking for when they're doing quality control in their pipeline runs. For example, she took what you get from MultiQC, so for example FastQC in this case, the first QC section. She drew little cartoony versions of what you should be seeing, separated by the three different boxes, but with little notes saying exactly what you are seeing in this and how you should interpret the results here. This acts as a very nice, quick and fast reference for people, to understand what your pipeline has done at the other end. Also it adds a little bit of fun, a little bit of color, to otherwise often very dry documentation, that can be often be very dull. Particularly for very big pipelines like Eager and Sarek, and when you have a lot of output documentation, such things can break things up a little bit and make it more accessible and more fun for a user to use.
```